x-pack/plugins/index_management/public/application/components/mappings_editor/__jest__/client_integration/helpers/setup_environment.tsx

```diff
@@ -83,6 +83,10 @@ const defaultProps = {
   docLinks: {
```

Comment:
I think it'd be better to replace this with the mock from the doc links service:

```
import {
  docLinksServiceMock,
  uiSettingsServiceMock,
} from '../../../../../../../../../../src/core/public/mocks';

...

const defaultProps = {
  docLinks: docLinksServiceMock.createStartContract(),
};

```